core/utils/pcap_handle.cc

```diff
@@ -0,0 +1,133 @@
+#include "pcap_handle.h"
+
+PCAPHandle::PCAPHandle() {
+  is_initialized_ = 0;
+  handle_ = nullptr;
+}
```

Comment:
No need for this here -- can just use an initializer list for the constructor in the .h file.